ams/server/src/test/java/com/netease/arctic/server/table/TestTableRuntimeHandler.java

```diff
@@ -66,12 +66,15 @@ public TestTableRuntimeHandler(
   }
 
   @Test
-  public void testInitialize() {
+  public void testInitialize() throws Exception {
     tableService = new DefaultTableService(new Configurations());
     TestHandler handler = new TestHandler();
     tableService.addHandlerChain(handler);
     tableService.initialize();
-    createDatabase();
+    if (!(catalogTestHelper().tableFormat().equals(TableFormat.MIXED_HIVE)
+        && TEST_HMS.getHiveClient().getDatabase(TableTestHelper.TEST_DB_NAME) != null)) {
+      createDatabase();
+    }
```

Comment:
Why adding this check?

Reply:
AMSTableTestBase::init  will  create the database  create test_db in advance. An error indicating the existence of the database will be reported here.